.gitignore

```diff
@@ -25,3 +25,6 @@ _testmain.go
 dist/
 
 dist/
+
+.vscode/
```

Comment:
Can I suggest you add `.vscode` to your global git ignore file, so that we won't have it here? Thanks!

Reply:
Good suggestion! I got it.